pkg/render/logstorage/linseed/linseed.go

```diff
@@ -305,7 +305,7 @@ func (l *linseed) linseedDeployment() *appsv1.Deployment {
 		)
 	}
 
-	if l.cfg.Tenant != nil {
+	if l.cfg.Tenant != nil && len(l.cfg.Tenant.Namespace) > 0 {
```

Comment:
This isn't quite right. We aren't checking if the namespace exists, we need to set the TENANT_NAMESPACE environment variable in Linseed's environment.

We will need to do the same for the other components that need this as well - Policy recommendation and Voltron